skbeam/core/accumulators/binned_statistic.py

```diff
@@ -316,9 +318,12 @@ def __call__(self, values, statistic=None):
                     null = np.nan
                 np.seterr(**old)
             self.result.fill(null)
-            for i in np.unique(self.xy):
+            p = Pool()
+
+            def op(i):
                 self.result[i] = statistic(values[self.xy == i])
 
+            [_ for _ in p.imap_unordered(op, np.unique(self.xy), 10)]
```

Comment:
We might consider moving the `self.xy == i` info into the `__init__` method to create a list of pixels in a bin, since this can be calculated from the initial information and would speed up computation as we don't need to recompute it.

Reply:
I'll see if I can get one in by COB Friday.